package.json

```diff
@@ -10,7 +10,8 @@
     "test": "jest --watchAll",
     "test:once": "jest",
     "build": "babel -d build --ignore __mocks__ src && node ./config/postbuild.js",
-    "serve": "node --harmony node_modules/sequelize-cli/lib/sequelize db:migrate --config build/config/database.js && node --harmony build/index.js"
+    "serve": "node --harmony node_modules/sequelize-cli/lib/sequelize db:migrate --config build/config/database.js && node --harmony build/index.js",
+    "blizztos": "nodemon --exec babel-node scripts/BlizzardTosDelete.js"
```

Comment:
Is `nodemon` necessary outside development?

Reply:
Unfortunately it is.  Running a script outside of the application otherwise requires a lot of workaround.  Using nodemon allows us to just run it with one command.  I'm open to options, but I didn't see another way.

Reply:
Just found a way to not use nodemon.  Didn't think I could.  Commit incoming.